README.md

```diff
@@ -212,43 +216,86 @@ https://beta.eyewire.org/2.0/tasks/assign?access_token=78q3ja8y
 
 ```json
 {
-  "id": 17444,
-  "seeds": [245, 5025, 6500],
-  "cell": 10,
+  "id": 558593,
```

Comment:
this change shouldn't be needed anymore. api is updated to fit the documentation